examples/html-css/index.html

```diff
@@ -5,20 +5,17 @@
     <title>Design System</title>
     <!-- In a real project this will contain a href link to your local CSS file, 
       while in this case we are using a CDN as an example -->
-    <link
-      rel="stylesheet"
-      href="https://unpkg.com/@cmsgov/design-system@2.0.0/dist/css/index.css"
-    />
+    <link rel="stylesheet" href="https://design.cms.gov/cdn/design-system/4.0.5/css/index.css" />
```

Comment:
Maybe we should add a `latest` symlink or something during CDN publish, just a thought. To be a future ticket perhaps.

Reply:
I like that idea!